docs/content/direct/release-testing.md

```diff
@@ -0,0 +1,31 @@
+# Testing a new KubeStellar Release
+
+The following testing process should be applied to every KubeStellar new release in order to validate the release, this include both regular releases and release candidates. All the tests should be done while the KubeStellar code is still under code-freeze and new code shouldn't be merged into the main branch until all tests are passed and the release is officially declared as ready.  
```

Comment:
s/ to every KubeStellar new release in order to validate the release/ to every new KubeStellar release in order to validate it/

Reply:
done